index.html

```diff
@@ -0,0 +1,1005 @@
+                    </div>
+                </div>
+            </div>
+            <div class="col-3">
+                <div class="card p-2 rounded h-100">
+                    <div class="d-flex align-items-center">
+                        <span class="tag">
+                            Servers
+                        </span>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">4. Scalable data use</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam a metus efficitur, porta lacus ac, pharetra nisl. Nulla non justo nec nibh rutrum ultrices sagittis in est.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-3">
+                <div class="card p-2 rounded h-100">
+                    <div class="d-flex align-items-center">
+                        <span class="tag">
+                            Applications
+                        </span>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">5. Flight booking application</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam a metus efficitur, porta lacus ac, pharetra nisl. Nulla non justo nec nibh rutrum ultrices sagittis in est.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-3">
+                <div class="card p-2 rounded h-100">
+                    <div class="d-flex align-items-center">
+                        <span class="tag">
+                            Applications
+                        </span>
+                    </div>
+                    <div class="card-body">
+                        <h3 class="card-title">6. Asset manager</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam a metus efficitur, porta lacus ac, pharetra nisl. Nulla non justo nec nibh rutrum ultrices sagittis in est.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="row text-center">
+            <div class="col-12">
+                <p>Want a more extensive read on our software structure?</p>
+                <a href="#" class="btn btn-secondary">Download the CONOPS</a>
+            </div>
+        </div>
+    </div>
+
+    <div class="container-fluid py-5 about bg-primary text-light">
+        <div class="container">
+            <div class="row d-flex align-items-center">
+                <div class="col-3 p-4"> <img class="rounded-circle img-teamlead w-100" src="/assets/img/photo-dummy.jpg" alt="FlyArrow team lead"></div>
+                <div class="col-9 ">
+                    <h3>"We are here to change the future of air mobility"</h3>
+                    <p class="special1">- AM Smith, team lead of FlyArrow.io</p>
```

Comment:
"VTOL cargo and passenger networks are the next economic gateway. We'll be able to forge new connections to rural communities without the financial and environmental costs of traditional ground and fixed-wing infrastructure. The impact this will have on commuting, supply chains, and humanitarian response can't be overstated."

A.M. Smith, software and systems engineer for FlyArrow.io

Reply:
fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b